boards/witte/linum/linum.dts

```diff
@@ -115,11 +108,21 @@
 };
 
 &pll2 {
-	div-m = <2>;
-	mul-n = <48>;
+	div-m = <5>;
+	mul-n = <96>;
 	div-p = <8>;
 	div-q = <40>;
-	div-r = <3>;
+	div-r = <10>;
+	clocks = <&clk_hse>;
+	status = "okay";
```

Comment:
This clock update is not documented. It would be helpfull to split i a dedicated commit to explain this change: why it is safe or required.